_topic_maps/_topic_map.yml

```
@@ -1261,6 +1261,12 @@ Topics:
   - Name: Disaster recovery considerations
     File: nbde-disaster-recovery-considerations
     Distros: openshift-enterprise,openshift-origin
+- Name: zero trust workload identity manager for Red Hat OpenShift
```

Is it always non-capitalized? I wouldn't think so, but I'm not 100% sure. 

```suggestion
- Name: Zero trust workload identity manager for Red Hat OpenShift
```